### ISSUE TITLE
Questions about the performance of iResNet-100 model on SMFRD in Table 5 of the paper

### ISSUE THREAD

**Issue by ClimberXIE** [zyainfal/Joint-Holistic-and-Masked-Face-Recognition#5]
Hi, 
I tested the IResNet100 model you released on SMFRD dataset, but the accuracy I get is only about 74%. I am not sure whether it is due to different test pairs. Could you please provide the test pair file of SMFRD?
Looking forward to your reply.
Best wishes.

**Comment by zyainfal** [comment]
Yes, the original SMFRD data is not well aligned and we re-aligned faces by RetinaFace, which is noted in the experiment part of the paper.

As to the data we used for experiments, we zipped and uploaded the file on Google Drive and Baidu Pan (please check readme for download)

**Comment by ClimberXIE** [comment]
Thank you for your prompt reply. I've checked the data. Our data is also aligned using Retinaface, and it looks the same as the data you provided, so I think it's due to different test pairs. But it seems you don't provide the test pairs file of SMFRD in the decompressed data?

**Comment by zyainfal** [comment]
Hi, 

we used pairs for LFW, CFP-FP, and AgeDB-30. 

The performance of SMFRD is the average of `accuracy4, accuracy5, and accuracy6` as in Line 463  to 467 in `trainer.py`, 
https://github.com/zyainfal/Joint-Holistic-and-Masked-Face-Recognition/blob/45394db77ba26afc4286095f5f7bd361770dbc15/code/trainer.py#L463-L467

therefore showing the impact of masked faces.

**Comment by ClimberXIE** [comment]
Thanks again for your reply, I have solved my problem now！
